src/pages/bubblegum-v2/concurrent-merkle-trees.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Concurrent Merkle Trees
-metaTitle: Concurrent Merkle Trees | Bubblegum
+metaTitle: Concurrent Merkle Trees | Bubblegum V2
 description: Learn more about Concurrent Merkle Trees and how they are used on Bubblegum.
```

Comment:
Seems like the advanced sections are the same, would be nice if they could point to the same files instead of be a copy

Reply:
They are. I've moved the V1 sections to V2 and the V1 ones just point you to V2.
There might be a better way to make it show in the menu - I'm open to suggestions

Reply:
They are - the bubblegum menu points to bubblegum v2 pages.